### ISSUE TITLE
Add a simple notification that an MMS was received

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#313]
```
In order to start with Issue 25, we should send a simple notification that an 
MMS was received.
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 15 Jan 2013 at 10:52
- Blocking: #25

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 15 Jan 2013 at 10:53
- Now blocking: #25

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 27 Jan 2013 at 1:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5a325833baf6.
```

Original comment by `yako...@gmail.com` on 28 Jan 2013 at 9:41
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
New version 4.4 Beta 1 is available in the download section
```

Original comment by `Florent....@gmail.com` on 16 Feb 2013 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
thanks for doing this! i'm looking forward to it. looks like it still hasn't 
made it into a release on the play store though. any plans to do that any time 
soon? i'd rather stick with the stable released version instead of beta, if 
possible.
```

Original comment by `heaven` on 1 May 2013 at 12:09

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4
```

Original comment by `Florent....@gmail.com` on 15 Sep 2013 at 3:27
- Changed state: **Deployed**
